### ISSUE TITLE
JAWS only reads table header

### ISSUE THREAD

**Issue by linda6907** [Mottie/tablesorter#690]
We are using tablesorter 2.15. 
JAWS only reads table header and reported the table has only one rows. And it doesn't read filter row and the rest of the table data rows. 

It seems work on the older version of tablesorter (2.4) on our another site.

Please help us.
Linda
